New version: TabulatedQuadratureRules v0.9.0

JuliaRegistrator opened the pull request:
- Registering package: TabulatedQuadratureRules
- Repository: https://github.com/quadraturerules/TabulatedQuadratureRules.jl
- Created by: @mscroggs
- Version: v0.9.0
- Commit: 672082b2a107ecc00f608c6105e5daf54506b7e0
- Reviewed by: @mscroggs
- Reference: https://github.com/quadraturerules/TabulatedQuadratureRules.jl/commit/672082b2a107ecc00f608c6105e5daf54506b7e0#commitcomment-165151609
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
- Added Newton–Cotes quadrature rules

## Breaking changes

- There are no breaking changes
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6d2e0e461d177eea2d7c158f4a94b5d38eb8263f38162b73f4540d6e63b8ae425bb0859cb7817986b9e1ae7a47ec241c7d827d054c6548ea1934ff1b021735c533dc0564f88ef77a6078238add76898a3f1a10ff4e0ef8bb246e0f1f1d3107b4ff263a28c20712b13ecb927eb5b280d646f5579067a4f0878079b5e2a7b9538fefc9191f9e0187be0f2da9c182c880e0040c70d5c85165770ca95ba86509a6fa78912d3ec555e17fcc5cfd43c480b2406fe6e0e30051e4024e99fabf05a3ac198 -->

mscroggs commented:
`[merge approved]`

mscroggs commented:
@JuliaRegistrator register

Release notes:

- Added Newton–Cotes quadrature rules

## Breaking changes

- There are no breaking changes
- ...

JuliaRegistrator committed:
New version: TabulatedQuadratureRules v0.9.0

UUID: 3da39775-d94e-428b-a709-24e9ab7fe27d
Repo: https://github.com/quadraturerules/TabulatedQuadratureRules.jl.git
Tree: ccb35f85512ea63893198040c0fe6f6f94b7aa67

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3
